hack/update-codegen.sh

```diff
@@ -109,6 +110,22 @@ function generate_group() {
     --output-file-base zz_generated.conversion \
     "$4"
 
+  echo "generating protocol buffers for group ${GROUP_NAME} and version ${VERSION} at ${SCRIPT_BASE}/${CLIENT_PKG}"
+
+  # The generated go_binaries are not guaranteed to be in exactly the
+  # corresponding package in bazel-bin; there may be another
+  # architecture-specific subdirectory.
+  # The call to xargs trims leading whitespace.
+  PROTOC_GEN_GOGO_PATH="$(dirname "$(bazel build //vendor/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo:protoc-gen-gogo 2>&1 | grep bazel-bin | xargs)")"
+  PROTOC_PATH="$(dirname "$(bazel build @com_google_protobuf//:protoc 2>&1 | grep bazel-bin | xargs)")"
+
+  PATH="$(bazel info workspace)/${PROTOC_PATH}:$(bazel info workspace)/${PROTOC_GEN_GOGO_PATH}:${PATH}" \
```

Comment:
Could we add a comment explaining that it's for the compiler to find the plugins?

Reply:
Done.